### PR TITLE
New SW Maint Pantry (Box) (like #17283 but more mergable)

### PR DESCRIPTION
**Pull request by coiax**
> First map change, might need some guidance.
> 
> I hate SW maint. All the side rooms have glass so all those solar engineers and whoever love to come gawk at you, just feels like there's not enough discrete locations like SE/NW/NE maint have.
> 
> So I moved an unremarkable little dead end around and gave us a new pantry!

:cl: Robustin
rscadd: A small pantry has been added to SW maintenance.
/:cl:

image

Robustin did the work, I'm just the guy with mapmerge set up.

I did one change, the loot spawn in the closet is now a 2maint loot spawn.

Implements #17283

### CODE REVIEW

**Comment by Bawhoppen** [comment]
>more mergeable

>fails travis

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -63994,39 +63999,38 @@
...
+/turf/open/floor/plating,
+/area/maintenance/aft)
 "cAE" = (
-/obj/structure/cable{
-	d1 = 1;
-	d2 = 2;
-	icon_state = "1-2"
+/obj/structure/table/glass,
+/obj/item/weapon/reagent_containers/food/condiment/saltshaker{
+	step_y = 2
```
Comment (lzimann) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: remove step or movement will break(use pixel_x/y)

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 2]
```diff
@@ -63994,39 +63999,38 @@
...
 	},
-/turf/open/floor/plasteel{
-	icon_state = "dark"
+/obj/item/weapon/reagent_containers/food/condiment/peppermill{
+	step_x = 2
```
Comment (lzimann) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: remove step or movement will break(use pixel_x/y)

**Comment by Robustin** [comment]
Did you fix the sink?

**Comment by coiax** [comment]
- [x] All access fridge (combined contents of pantry and fridge)
- [x] Moved sink
- [x] Added mint

**Comment by ChangelingRain** [comment]
ew that sink isn't aligned right add some pixel shift to it

**Comment by ChangelingRain** [comment]
frankly I think this should have less lights but whateveeeer

**Pull request merged**